lib/components/src/Accordion/stories/AccordionStories.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import type { Meta, Story } from '@storybook/react';
+import React from 'react';
+import { Accordion } from '../Accordion';
+import { AccordionBody } from '../AccordionBody';
+import { AccordionHeader } from '../AccordionHeader';
+import { AccordionItem } from '../AccordionItem';
+
+export default {
+  title: 'Basics/Accordion/Stories',
+  component: Accordion,
+  parameters: {
+    actions: { disable: true },
+    test: { disable: true },
+    controls: { disable: true },
+  },
+} as Meta;
+
+export const OneDefaultOpen: Story = () => (
+  <Accordion open={1}>
+    <AccordionItem>
+      <AccordionHeader>Item 1 - Default open</AccordionHeader>
+      <AccordionBody>
+        Minim proident eu aliqua irure tempor incididunt fugiat. Adipisicing aliquip cillum esse
+        amet. Consequat qui consectetur duis laboris aliqua fugiat Lorem eiusmod ut cupidatat
+        excepteur. Magna nulla nulla velit voluptate duis nulla quis Lorem dolore labore aliqua sit
+        ipsum.
+      </AccordionBody>
+    </AccordionItem>
+    <AccordionItem>
+      <AccordionHeader>Item 2</AccordionHeader>
+      <AccordionBody>
+        Minim proident eu aliqua irure tempor incididunt fugiat. Adipisicing aliquip cillum esse
+        amet. Consequat qui consectetur duis laboris aliqua fugiat Lorem eiusmod ut cupidatat
+        excepteur. Magna nulla nulla velit voluptate duis nulla quis Lorem dolore labore aliqua sit
+        ipsum.
+      </AccordionBody>
+    </AccordionItem>
+    <AccordionItem>
+      <AccordionHeader>Item 3</AccordionHeader>
+      <AccordionBody>
+        Minim proident eu aliqua irure tempor incididunt fugiat. Adipisicing aliquip cillum esse
+        amet. Consequat qui consectetur duis laboris aliqua fugiat Lorem eiusmod ut cupidatat
+        excepteur. Magna nulla nulla velit voluptate duis nulla quis Lorem dolore labore aliqua sit
+        ipsum.
+      </AccordionBody>
+    </AccordionItem>
+    <AccordionItem>
+      <AccordionHeader>Item 4</AccordionHeader>
+      <AccordionBody>
+        Minim proident eu aliqua irure tempor incididunt fugiat. Adipisicing aliquip cillum esse
+        amet. Consequat qui consectetur duis laboris aliqua fugiat Lorem eiusmod ut cupidatat
+        excepteur. Magna nulla nulla velit voluptate duis nulla quis Lorem dolore labore aliqua sit
+        ipsum.
+      </AccordionBody>
+    </AccordionItem>
+  </Accordion>
+);
+
+export const MultipleDefaultOpen: Story = () => (
+  <Accordion open={[1, 2]} allowMultipleOpen>
```

Comment:
I think we should EITHER:
- allow users to specify `open` and have THEM manage the state.
- allow users to specify `initial` and have US manage the state.

but not a mix of the 2.

Preferably those 2 use-cases are encapsulated into separate components.

Reply:
Why not both?

The way it is done now you can do it all, in every use case there can arise